es-app/src/views/gamelist/ISimpleGameListView.h

```diff
@@ -32,7 +32,7 @@ class ISimpleGameListView : public IGameListView
 	ImageComponent mHeaderImage;
 	ImageComponent mBackground;
 
-	std::vector<GuiComponent*> mThemeExtras;
```

Comment:
The `ThemeExtras` class was removed in the time since this grid view implementation was originally done.  This change needs to be reverted.